.github/workflows/container.yaml

```diff
@@ -0,0 +1,56 @@
+# See https://docs.earthly.dev/ci-integration/vendor-specific-guides/gh-actions-integration
+# for details.
+
+name: Build container
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  push:
+    branches:
+      - webv2
+  pull_request:
+    branches:
+      - webv2
+
+jobs:
+  Earthly:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Earthly
+        uses: earthly/actions-setup@v1
+        with:
+          version: 'latest' # or pin to an specific version, e.g. "v0.6.10"
+
+      - name: Log Earthly version
+        run: earthly --version
+
+      - name: Authenticate to GitHub Container Registry
+        if: ${{ github.event_name == 'schedule' && env.GH_CR_PAT != null }}
+        env:
+          GH_CR_PAT: ${{ secrets.GH_CR_PAT }}
+        run: echo "${{ secrets.GH_CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          image: tonistiigi/binfmt:latest
+          platforms: all
+
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build and push
+        if: ${{ github.event_name == 'schedule' && env.GH_CR_PAT != null }}
+        env:
+          GH_CR_PAT: ${{ secrets.GH_CR_PAT }}
+          EARTHLY_BUILD_TAG: 'webv2'
```

Comment:
If this is going to be merged into `webv2` then the builds in `develop` are going to keep running. Those jobs won't get overwritten until `webv2` get merged into `develop`, and when that happens there will be no `webv2` branch anymore. So we should either plan for when that merge happens, or plan for now when both nightly and webv2 builds are happening. This PR kind of is in a middle ground.

Reply:
It is in the list for #2437
After the merge, all references to `webv2` in workflow files must be replaced by `develop`

Reply:
But before the merge these workflows won't even run, will they? Don't they need to be in the primary branch to run? Or that's what I assumed, I could be wrong I've never tried it. That's why I don't have `webv2` nightlies firing from the `webv2` branch.

Reply:
These should get triggered for the PRs to `webv2` too. (as they have ran for this PR) Only the deployment to GH part is skipped.